gremlin/templates/_helpers.tpl

```diff
@@ -30,3 +30,19 @@ Create chart name and version as used by the chart label.
 {{- define "gremlin.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{/*
```

Comment:
I'm actually not sure if this block works yet, I need to test it, but wanted to share anyway in case I'm way off target